python/doc/demos/NODDI_turorial_01.md

```diff
@@ -0,0 +1,124 @@
+# Tutorial for python-AMICO/NODDI
+
+Here you can learn how to run python-AMICO/NODII for sampel dateset.
+
+## 1. Prepare dataset
+
+The sample dataset is available from [NODDI official website](http://mig.cs.ucl.ac.uk/index.php?n=Tutorial.NODDImatlab).
+
+After unzip, make the directory strucutre as followings:
+
+```
+└── Study01
+    └── Subject01
+        ├── NODDI_DWI.hdr
+        ├── NODDI_DWI.img
+        ├── NODDI_protocol.bval
+        ├── NODDI_protocol.bvec
+        ├── brain_mask.hdr
+        ├── brain_mask.img
+        ├── roi_mask.hdr
+        └── roi_mask.img
+```
+
+## 2. DWI preprocess
+
+Usually you need some preprocesses for DWI images (e.g., eddy current correction, head movement correction, and skull stripping).
+You need do these preprocesses before performing python-AMICO/NODDI.
+
+Assuming these preprocesses have been done for the sample dataset, we skip this step here.
+
+## 3. Performe python-AMICO/NODDI
+
+Now, you are in the directory where `Study01` exists.
+
+```bash
+$ ls
+Study01
+```
+
+Then, run python (of course, ipython is your best friend),
+
+```bash
+$ python
+```
+
+Then, in python, you can perform python-AMICO/NODDI:
```

Comment:
Why, instead of a single snippet with code, comments and output, don't we split it into "sections" with description and code where we explain what we do in each section?

Example:

Setup AMICO:
```python
import amico
amico.core.setup()
```

Create the scheme file from the existing bvals/bvecs using the `fsl2scheme` in `amico.util`:
```python
>>> amico.util.fsl2scheme("Study01/Subject01/NODDI_protocol.bval", "Study01/Subject01/NODDI_protocol.bvec")
-> Writing scheme file to [ Study01/Subject01/NODDI_protocol.scheme ]
```
etc etc